data/json/monsters/mammal.json

```
@@ -2351,5 +2351,60 @@
     "death_function": [ "NORMAL" ],
     "zombify_into": "mon_zolf",
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE" ]
+  },
+  {
+    "id": "mon_reindeer",
+    "type": "MONSTER",
+    "name": { "str": "Reindeer" },
+    "description": "Also known as the Caribou, this deer is native to North America and parts of Eurasia, although usually friendly, they can be aggressive during mating season as well as when hungry.",
+    "default_faction": "herbivore",
+    "bodytype": "horse",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "889000 ml",
+    "weight": "240 kg",
+    "hp": 120,
+    "speed": 325,
+    "material": [ "flesh" ],
+    "symbol": "R",
+    "color": "brown",
+    "aggression": 7,
+    "morale": 82,
+    "melee_skill": 3,
+    "melee_dice": 3,
+    "melee_dice_sides": 3,
+    "melee_cut": 0,
+    "dodge": 5,
+    "vision_night": 13,
+    "harvest": "mammal_large_leather",
+    "path_settings": { "max_dist": 14 },
+    "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
+    "anger_triggers": [ "HURT", "MATING_SEASON" ],
+    "death_function": [ "NORMAL" ],
+    "zombify_into": "mon_zeindeer",
+    "reproduction": { "baby_monster": "mon_reindeer_fawn", "baby_count": 5, "baby_timer": 340 },
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 12 },
+    "baby_flags": [ "WINTER", "AUTUMN" ],
+    "special_attacks": [ [ "EAT_CROP", 60 ] ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "CATTLEFODDER", "PET_MOUNTABLE" ]
```

Wild animals should not be tameable.
```suggestion
    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM" ]
```